core/server/worker/src/test/java/alluxio/worker/block/DefaultBlockWorkerTest.java

```diff
@@ -180,9 +181,9 @@ public void commitBlockOnRetry() throws Exception {
   public void createBlock() throws Exception {
     long blockId = mRandom.nextLong();
     long sessionId = mRandom.nextLong();
-    long initialBytes = 1;
+    long initialBytes = 1024 * 1024 * 1024;
```

Comment:
Can you use Constants.MB? Can we not initialize a large block store  e.g. https://github.com/Alluxio/alluxio/pull/13375?

Reply:
done